.travis.yml

```diff
@@ -7,9 +7,18 @@ branches:
     - master
     - dev
 
+before_install:
+    - sudo add-apt-repository -y ppa:snaipewastaken/ppa
+    - sudo apt-get update -q
+    - sudo apt-get install criterion-dev -y
+
 jobs:
   include:
     - stage: build
-      script: echo Travis # dummy command until we have make files
+      script: make clean all
```

Comment:
You don't need to `make clean` here - each Travis build starts from a fresh environment and a fresh clone of your code. It's not wrong but it's not necessary either.